apps/nextra/pages/en/build/indexer.mdx

```diff
@@ -33,52 +36,89 @@ Here's an interactive example of how you can query for the Current Coin Balances
 }`}
 />
 
+<Callout type="info">
+The Indexer tracks every transaction that happens on-chain, then exposes that data through a GraphQL API.
+</Callout>
+
+## Using the Indexer API
 
-## Indexer API
+Learn how to use the Indexer API, what each table represents, and how to add custom data to the Indexer.
 
 <Cards>
-  <Card href="indexer/api/labs-hosted">
-    <Card.Title>Aptos Labs Indexer API</Card.Title>
-    <Card.Description>Query Aptos Labs GraphQL Indexer API hosted on Hasura</Card.Description>
+  <Card href="indexer/aptos-hosted">
+    <Card.Title>Accessing the API</Card.Title>
+    <Card.Description>Learn how to query the Indexer API.</Card.Description>
   </Card>
-  <Card href="indexer/api/self-hosted">
-    <Card.Title>Self-hosted Indexer API</Card.Title>
-    <Card.Description>Host your own Indexer API</Card.Description>
+    <Card href="indexer/indexer-reference">
+    <Card.Title>Indexer Table Reference</Card.Title>
+    <Card.Description>Detailed reference for Indexer tables and their schemas.</Card.Description>
   </Card>
   <Card href="indexer/architecture">
     <Card.Title>Architecture</Card.Title>
-    <Card.Description>Detailed layout of Indexer Architecture</Card.Description>
-  </Card>
-  <Card href="indexer/custom-processors">
-    <Card.Title>Indexer Processors</Card.Title>
-    <Card.Description>Use existing or custom indexer processors to shape on-chain data for your application</Card.Description>
+    <Card.Description>Detailed layout of the Indexer's architecture.</Card.Description>
   </Card>
 </Cards>
 
-## Example Queries
+### Example Queries
+
+To help get you started, here are the most common queries the Indexer is used for.
 
 <Cards>
+  <Card href="indexer/coin-balances">
+    <Card.Title>Get Coin Balances</Card.Title>
+    <Card.Description>Get all coins an account currently owns.</Card.Description>
+  </Card>
   <Card href="indexer/account-transactions">
-    <Card.Title>Account Transactions Query</Card.Title>
-    <Card.Description>Retrieves `transaction_version`s of transactions that affect a specified account address, ordered in descending order</Card.Description>
+    <Card.Title>Get Account Transactions</Card.Title>
+    <Card.Description>Get all transactions impacting an account.</Card.Description>
   </Card>
-  <Card href="indexer/coin-balances">
-    <Card.Title>Coin Balances Query</Card.Title>
-    <Card.Description>Retrieves the current balances for various coin types associated with a specified account address</Card.Description>
+  <Card href="indexer/ans-lookup">
+    <Card.Title>Get Account Domain Name</Card.Title>
```

Comment:
```suggestion
    <Card.Title>Get Aptos Name</Card.Title>
```
I haven't seen this called Account Domain Name anywhere (I might be wrong though)